src/codeu/chat/server/Server.java

```diff
@@ -190,6 +204,23 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
       public void run() {
         try {
 
+          try {
+            // writes to this persistentpath textfile and will append all new information to the end of the existing file
+            persistentDataWriter = new PrintWriter(new FileWriter(persistentPath + "/persistent_data.txt", true));
```

Comment:
for some reason I don't get that error.. is it because I'm on a mac and the folder slashes are different? Should I create a check for this (is this an example of what a test would have checked against?)